flexmeasures/data/models/planning/storage.py

```diff
@@ -207,12 +211,15 @@ def _prepare(self, skip_validation: bool = False) -> tuple:
             StorageScheduler.COLUMNS, start, end, resolution
         )
 
-        ems_power_capacity_in_mw = self.flex_model.get(
+        ems_power_capacity_in_mw = self.flex_context.get(
             "ems_power_capacity_in_mw",
             self.sensor.generic_asset.get_attribute("capacity_in_mw", None),
         )
 
         if ems_power_capacity_in_mw is not None:
+            if isinstance(ems_power_capacity_in_mw, ur.Quantity):
+                ems_power_capacity_in_mw = ems_power_capacity_in_mw.magnitude
```

Comment:
When might this happen?

Reply:
This happens when the attributes comes from the flex-model, it is serialized as a Quantity object.